.github/workflows/R-CMD-check.yaml

```diff
@@ -5,6 +5,7 @@ on:
     branches:
       - main
       - master
+      - dev/vignette_fix
```

Comment:
this can come out (looks like from @OJWatson :grin: )

Reply:
done